packages/frontend/src/components/attachment/messageAttachment.tsx

```diff
@@ -54,6 +55,22 @@ export default function Attachment({
       openAttachmentInShell(message)
     }
   }
+  /**
+   * we need a fix height for images to prevent reflow when the image loads
+   * till we have a better solution we allow a maximum height of 200px for
+   * all images in landscape mode and 300px for portrait mode
+   *
+   * for higher values we have to add media queries otherwise messages with
+   * images get strange paddings on small windows
```

Comment:
For higher values of what? Aspect ratio?

Reply:
Higher than 200px. The problem is, that with fixed height once the message column gets too narrow the message keeps the initial height while the image keeps it ratio and is thus not filling the height of the message any more - which results in the mentioned paddings